clang/lib/AST/ExprUtils.cpp

```diff
@@ -469,7 +469,7 @@ void ExprUtil::EnsureEqualBitWidths(llvm::APSInt &A, llvm::APSInt &B) {
 }
 
 bool InverseUtil::IsInvertible(Sema &S, Expr *LValue, Expr *E) {
-  if (!E)
+  if (!E || E->containsErrors())
```

Comment:
Maybe we should add the same check in `InverseUtil::Inverse(...)`?

Reply:
Should we do this on a case-by-case basis?

Reply:
Yes. But we could think of functions `IsInvertible` and `Inverse` as the same "case". Though `Inverse` is called only when `IsInvertible` returns true, it may be safe to add the check in the function `Inverse` also.